com.jetbrains.RustRover.yml

```diff
@@ -0,0 +1,141 @@
+app-id: com.jetbrains.RustRover
+command: rustrover
+runtime: org.freedesktop.Sdk
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+separate-locales: false
+tags:
+  - proprietary
+
+finish-args:
+  - --allow=devel
+  - --device=all
+  - --env=RUST_ROVER_PROPERTIES=/app/bin/idea.properties
+  - --filesystem=host
+  - --filesystem=xdg-run/app/com.discordapp.Discord:create
+  - --filesystem=xdg-run/docker
+  - --filesystem=xdg-run/gnupg
+  - --filesystem=xdg-run/keyring
+  - --filesystem=xdg-run/podman
+  - --share=ipc
+  - --share=network
+  - --socket=gpg-agent
+  - --socket=ssh-auth
+  - --socket=wayland
+  - --socket=x11
```

Comment:
it's either wayland + fallback-x11 or just x11.

Reply:
PR updated

Reply:
Looks like it needs access to x11